strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/local/RepositoryHostedIndexCreator.java

```diff
@@ -183,17 +191,17 @@ private ArtifactEntryArtifactContextHelper createArtifactContextHelper(final Art
     /**
      * org.apache.maven.index.DefaultArtifactContextProducer#isIndexable(java.io.File)
      */
-    private boolean isIndexable(final ArtifactEntry artifactEntry)
+    private boolean isIndexable(final Artifact artifactEntry)
     {
         final String filename = Paths.get(artifactEntry.getArtifactPath()).getFileName().toString();
 
         if (filename.equals("maven-metadata.xml")
-            // || filename.endsWith( "-javadoc.jar" )
-            // || filename.endsWith( "-javadocs.jar" )
-            // || filename.endsWith( "-sources.jar" )
-            || filename.endsWith(".properties")
-            // || filename.endsWith( ".xml" ) // NEXUS-3029
-            || filename.endsWith(".asc") || filename.endsWith(".md5") || filename.endsWith(".sha1"))
+                // || filename.endsWith( "-javadoc.jar" )
+                // || filename.endsWith( "-javadocs.jar" )
+                // || filename.endsWith( "-sources.jar" )
+                || filename.endsWith(".properties")
+                // || filename.endsWith( ".xml" ) // NEXUS-3029
+                || filename.endsWith(".asc") || filename.endsWith(".md5") || filename.endsWith(".sha1"))
```

Comment:
This looks like a non-change, (unless I'm overlooking something)? If so, could you please revert it? Thanks! :)

Reply:
reverted

Reply:
Thanks!